web-services-sample-app/ShowWeatherVC.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ShowWeatherVC.swift
+//  web-services-sample-app
+//
+//  Created by webonise on 29/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ShowWeatherVC: UIViewController {
+    
+    @IBOutlet var textFieldLatitude: UITextField!
+    @IBOutlet var textFieldLongitude: UITextField!
+    @IBOutlet var textFieldMaxTemperature: UITextField!
+    @IBOutlet var textFieldMinTemperature: UITextField!
+    var enteredCityName: String?
+    var weather: Weather = Weather()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(true)
+        self.webServicesRequest()
+    }
+    
+    /// This function calls api and converts the data of json format into NSDictionary. The function accepts a url and calls api to fetch the information of JSON type into responseString. Then JSON document is converted into NSDictionary type.
+    ///- returns: Void
+    func webServicesRequest() -> Void {
+        let scriptUrl = CommonConstants.WebServices.URL
+        let urlWithParams = scriptUrl + enteredCityName! +  CommonConstants.WebServices.AND + CommonConstants.WebServices.APPID +  (CommonConstants.WebServices.APPID_VALUE)
+        let myUrl = NSURL(string: urlWithParams.stringByAddingPercentEscapesUsingEncoding(NSUTF8StringEncoding)!)
+        let request = NSMutableURLRequest(URL:myUrl!);
+        request.HTTPMethod = CommonConstants.WebServices.GET_REQUEST
+        let task = NSURLSession.sharedSession().dataTaskWithRequest(request) {
+            data, response, error in
+            if error != nil
+            {
+                print("error=\(error)")
+                return
+            }
+            let responseString = NSString(data: data!, encoding: NSUTF8StringEncoding)
+            print("responseString = \(responseString)")
+            do {
+                    if let convertedJsonIntoDict = try NSJSONSerialization.JSONObjectWithData(data!, options: []) as? NSDictionary
+                    {
+                        print(convertedJsonIntoDict)
+                        self.parseDictionary(convertedJsonIntoDict)
+                    }
+                }
+                
+             catch let error as NSError {
+                print(error.localizedDescription)
+            }
+        }
+        task.resume()
+    }
+   
+    /// This function stores value of specific key in separate variables. The local variables are then converted to string to store them into model's object. The function also calls the displayWeatherDetails() function
+    ///- parameter: dict of NSDictionary type
```

Comment:
Documentation format is not right, refer this and make corrections
/**
     Method returns the count for each sport
     - parameter section: denotes the sport index
     - returns:Int,the count of the books for that sport
     */